napari/layers/labels/_tests/test_labels.py

```diff
@@ -1871,3 +1871,17 @@ def test_new_colormap_int8():
     data = np.arange(-128, 128, dtype=np.int8).reshape((16, 16))
     layer = Labels(data)
     layer.new_colormap(seed=0)
+
+
+@pytest.mark.parametrize('visible', [True, False])
+@pytest.mark.parametrize('dtype', [np.uint8, np.uint32, np.int64])
```

Comment:
Given your [comments](https://github.com/Czaki/napari/pull/40#issuecomment-2850097112) @Czaki, why not include int8 in this list?

Reply:
```suggestion
@pytest.mark.parametrize('dtype', [np.uint8, np.int8, np.uint32, np.int64])
```

No reason